### ISSUE TITLE
Header. GC signature bilingual alt text is incomprehensible to voice over

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#129]
### Description of issue
With voiceover, the GC signature alt text contains both French and English, regardless of the current language. The alt text should be functional rather than reproduce word for word what text is contained in the logo. 

For an anglophone user, sufficient alt text should be "Government of Canada" 
For francophone users, sufficient alt text would be "Gouvernement du Canada"

### Potential fix
- Only use one language in the logo alt text

### Resources
https://www.deque.com/blog/great-alt-text-introduction/

**Comment by amazingphilippe** [comment]
This has been fixed.
